packages/babel-plugin-import-jsx-pragma/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 2.0.0 (Unreleased)
+
+### Breaking Change
+
+- Plugin skips now adding import JSX pragma when the scope variable is defined for all JSX elements ([#13809](https://github.com/WordPress/gutenberg/pull/13809)). 
```

Comment:
1. I'm having a hard time interpreting this.
2. Is it really a breaking change? I'm having a hard time thinking how my code could possibly break after the changes here.

Reply:
I think you are right. In that sense, all code should work as before. It's definitely some kind of change but I probably went too far and I tried to be overcautious. I will update to `New Feature` and minor update.

Reply:
It's hard to be honest to decide, you could also consider it a bug fix :)

Reply:
> I will update to `New Feature` and minor update.

It was never updated? I think enhancement makes sense, yes.

Reply:
Hmm, something went wrong during rebase... I also overrode your commit. I will open a follow-up PR. Thanks for catching 👍

Reply:
#14106 opened to fix everything, sorry about that, I still don't know how it has happened ... :(